README.md

```diff
@@ -34,17 +75,17 @@ complicated. Especially if you don't implement too many features.
 | Key   | Description                                            |
 |-------|--------------------------------------------------------|
 | Space | Show the room switcher.                                |
-| j*    | Select one line down.                                  | 
-| k*    | Select one line up.                                    | 
-| i     | Create a new message using the external editor.        | 
-| Enter | Open the selected message (images, videos, urls, etc). | 
-| s     | Save the selected message (images and videos).         | 
-| c     | Edit the selected message in the external editor.      | 
-| r     | React to the selected message.                         | 
-| R     | Reply to the selected message.                         | 
-| v     | View the selected message in the external editor.      | 
-| V     | View the current room in the external editor.          | 
-| u     | Upload a file.                                         | 
+| j*    | Select one line down.                                  |
```

Comment:
Sorry about the overactive formatter here trimming whitespace. If it's annoying I can add them back and save without formatting.